orchestra/static/orchestra/task/js/services.js

```diff
@@ -47,9 +47,26 @@ serviceModule.factory('orchestraService', function() {
         }
     }
 
+    var signals = {
+        // Functions to override in specific task controllers.
+        'save': {
```

Comment:
Sounds good to me—I like the `registerSignal('submit.before', function() {...})` style, so I'll take a stab at that now!